docs/libcurl/opts/CURLOPT_TCP_NODELAY.3

```diff
@@ -52,7 +52,7 @@ CURL *curl = curl_easy_init();
 if(curl) {
   curl_easy_setopt(curl, CURLOPT_URL, "http://example.com");
   /* disable Nagle */
-  curl_easy_setopt(curl, CURLOPT_TCP_FASTOPEN, 0);
```

Comment:
Typo in the typo fix! =) The option is called `CURLOPT_TCP_NODELAY`...

Reply:
LOL fixed